server/experimental/src/common/getopt.cpp

```diff
@@ -0,0 +1,188 @@
+/*****************************************************************************
```

Comment:
gflags lib already have function like/better than this - let's try to use gflags.

Reply:
I will look into it.

Reply:
I looked into it and think it is too heavy for the temporary purpose of command line driver, which only needs to be told if the input is from a file or should it become interactive. I can get rid of getopt and let andb.exe become interactive if argv[1] is NULL, or treat argv[1] as input file, which will avoid the issue of command line argument processing.